classes/services/PKPSchemaService.php

```diff
@@ -67,6 +67,8 @@ class PKPSchemaService
      */
     public function get($schemaName, $forceReload = false)
     {
+        Hook::call('Schema::get::before::' . $schemaName, [&$forceReload]);
```

Comment:
Use `Hook::run` for new hooks, and improved calling conventions for hook callback functions. See https://github.com/pkp/pkp-lib/discussions/8089 for details.

Reply:
Right - fixed